layouts/partials/deprecation-warning.html

```diff
@@ -5,7 +5,7 @@ <h3>
       {{ T "deprecation_title" }} {{ .Param "version" }}
     </h3>
     <p> Kubernetes {{ .Param "version" }} {{ T "deprecation_warning" }}
-      <a href="{{ site.Params.currentUrl }}">{{ T "latest_version" }}</a>
+      <a href="{{ site.Params.currentUrl }}{{ .RelPermalink }}">{{ T "latest_version" }}</a>
```

Comment:
This is creating an additional issue. Since the `currentUrl` param is set to `https://kubernetes.io/docs/home/`, the link then becomes https://kubernetes.io/docs/home//blog/ for the blog page for eg. This is an invalid link.

Reply:
Good spot. Maybe we should change `site.Params.currentUrl ` or use another means to figure out the latest one. I don't think the `/docs/home/` part is expected there.

Reply:
Doesn't look like either `currentUrl` is used anywhere apart from the deprecation partial. Even `nextUrl` doesn't seem to be used, unless they are used in some script outside this repo. In which case, we should verify changing `currentUrl` doesn't break any workflow.